jax/experimental/ode.py

```diff
@@ -193,17 +196,17 @@ def body_fun(state):
     return carry, y_target
 
   f0 = func_(y0, ts[0])
-  dt = initial_step_size(func_, ts[0], y0, 4, rtol, atol, f0)
+  dt = init_step
   interp_coeff = np.array([y0] * 5)
   init_carry = [y0, f0, ts[0], dt, ts[0], interp_coeff]
   _, ys = lax.scan(scan_fun, init_carry, ts[1:])
   return np.concatenate((y0[None], ys))
 
-def _odeint_fwd(func, rtol, atol, mxstep, y0, ts, *args):
-  ys = _odeint(func, rtol, atol, mxstep, y0, ts, *args)
+def _odeint_fwd(func, rtol, atol, init_step, mxstep, y0, ts, *args):
+  ys = _odeint(func, rtol, atol, init_step, mxstep, y0, ts, *args)
   return ys, (ys, ts, args)
 
-def _odeint_rev(func, rtol, atol, mxstep, res, g):
+def _odeint_rev(func, rtol, atol, init_step, mxstep, res, g):
```

Comment:
Do we want to automatically reuse the same initial step size for the reverse pass?

It seems like we might want to recalculate the proper step size, at least in the case where `init_step=None`.

Reply:
Ah yes, didn't think about this. I personally don't have a use case for manually setting the `init_step` for the reverse pass. It also seems a little too involved to implement, especially since in general the user should be allowed to specify `init_step` for each of the observation points on the backwards pass.

For these reasons, I propose to always automatically choose the step size on the backwards pass. What do you think?

Reply:
> For these reasons, I propose to always automatically choose the step size on the backwards pass. What do you think?

Sounds good to me